bin/run_pypit

```diff
@@ -83,11 +88,11 @@ def main() :
 
     # Execute the reduction, and catch any bugs for printout
     if debug['develop']:
-        pypit.PYPIT(red, progname=pypit.__file__, quick=qck, ncpus=cpu, verbose=vrb,
+        pypit.PYPIT(red, initmsgs, progname=pypit.__file__, quick=qck, ncpus=cpu, verbose=vrb,
```

Comment:
Upon further review, I've done some mixing about.  
Didn't make sense to me to pass around version and last_updated.